apps/api/config/prod.exs

```diff
@@ -15,8 +15,10 @@ use Mix.Config
 # which you typically run after static files are built.
 config :api, API.Web.Endpoint,
   on_init: {API.Web.Endpoint, :load_from_system_env, []},
-  url: [host: "example.com", port: 80],
-  cache_static_manifest: "priv/static/cache_manifest.json"
+  url: [scheme: "https", host: "zombees.herokuapp.com", port: 443],
+  force_ssl: [rewrite_on: [:x_forwarded_proto]],
+  cache_static_manifest: "priv/static/cache_manifest.json",
```

Comment:
Do we need this? Seems like we wouldn't since we're not serving any static assets